test/e2e/util.go

```diff
@@ -481,9 +482,14 @@ func kubectlCmd(args ...string) *exec.Cmd {
 	}
 	kubectlArgs := append(defaultArgs, args...)
 
-	//TODO: the "kubectl" path string might be worth externalizing into an (optional) ginko arg.
-	cmd := exec.Command(filepath.Join(testContext.RepoRoot, "cluster/kubectl.sh"), kubectlArgs...)
-	Logf("Running '%s %s'", cmd.Path, strings.Join(cmd.Args, " "))
+	//Allow users to specify KubectlPath at commandline... so that we can test kubectl (the binary)
+	//Or so that, in the dev mode, we can test the kubectl shell script.
+	cmdToRun := testContext.KubectlPath
```

Comment:
hi ixdy.  

Good eye :) Here is my thinking: 

We *can* do RepoRoot append if  you are okay with an "empty" or "nil" default value for 
- KubectlPath which serves as an indicator that we need to dynamically generate the path from RepoRoot.
- current comments for RepoRoot indicate that its mainly for `examples` and this seemed more like a temporary solution. 

Im okay either way  , just wanted to make sure you realized this tradeoff .  let me know your thoughts and ill implement.